internal/net/net.go

```diff
@@ -36,30 +38,38 @@ const (
 	defaultPort = 80
 )
 
-type Conn = net.Conn
-type Dialer = net.Dialer
-type ListenConfig = net.ListenConfig
-type Listener = net.Listener
-type Resolver = net.Resolver
+type (
+	Conn         = net.Conn
+	Dialer       = net.Dialer
+	ListenConfig = net.ListenConfig
+	Listener     = net.Listener
+	Resolver     = net.Resolver
```

Comment:
Could you please add comment for each type? there are global.

Reply:
thank you for your comment. how about this?
https://github.com/vdaas/vald/pull/615/files#r469718324

Reply:
seems good